src/service/video-manager-impl.js

```diff
@@ -317,14 +317,23 @@ export class VideoManager {
   }
 
   /**
-   * Gets the current analytics details for the given video.
+   * Gets the current analytics details property for the given video.
    * Fails silently if the video is not registered.
-   * @param {!AmpElement} videoElement
+   * @param {string} id
+   * @param {string} property
    * @return {!Promise<!VideoAnalyticsDetailsDef|undefined>}
    */
-  getAnalyticsDetails(videoElement) {
+  getAnalyticsDetail(id, property) {
```

Comment:
nit: maybe rename to `getVideoState` to disambiguate from the other object?

Reply:
dang, beat me to it 👍